session.py

```diff
@@ -25,15 +26,18 @@
 """
 
 
-
-
 class GaeSession:
-	plainCookieName = "viurHttpCookie"
-	sslCookieName = "viurSSLCookie"
+	"""Store Sessions inside the Big Table/Memcache"""
 	kindName = "viur-session"
 	sameSite = "lax"
 
-	"""Store Sessions inside the Big Table/Memcache"""
+	def __init__(self):
+		super().__init__()
+
+		ident = os.environ.get("GOOGLE_CLOUD_PROJECT", "")
```

Comment:
Should use projectID from core.utils here, otherwise okay for me

Reply:
you're right, I've just fixed it, using `utils.projectID`, and an underscore before.